mitosheet/src/types.tsx

```diff
@@ -723,6 +727,11 @@ export interface UserProfile {
     mitoConfig: {
         [MitoEnterpriseConfigKey.MEC_VERSION]: number | undefined
         [MitoEnterpriseConfigKey.SUPPORT_EMAIL]: string
+        [MitoEnterpriseConfigKey.CODE_SNIPPETS]: {
+            [MitoEnterpriseConfigKey.CODE_SNIPPETS_VERSION]: string | undefined,
+            [MitoEnterpriseConfigKey.CODE_SNIPPETS_URL]: string | undefined
+            [MitoEnterpriseConfigKey.CODE_SNIPPETS_SUPPORT_EMAIL]: string
```

Comment:
I see that point for sure. But I didn't take this update because I wanted to give users helpful error messages when they try to open the code snippets taskpane. If they have only defined some of the variables, then we give them helpful guidance on how to correct it. 

And since we don't use these variables in the frontend except for CODE_SNIPPETS_SUPPORT_EMAIL, I think it's fine for now. Another option would be:

Moving all of the environment variable error handling into the MITO_CONFIG file itself so the errors are reported right away instead of when the user opens the code snippets taskpane and then having all of the config variables in the step performer being validated and ready to go. 

But potentially that is worse for end users who would see those errors right away even if they don't use the features that are not configured properly. + there is not a lot that the end user can actually do about it besides go ask their IT team to fix.